addon/components/single-event.js

```diff
@@ -3,10 +3,9 @@ import { computed } from '@ember/object';
 import { reads } from '@ember/object/computed';
 import { inject as service } from '@ember/service';
 import { isBlank, isEmpty } from '@ember/utils';
-import EventMixin from 'ilios-common/mixins/events';
 import moment from 'moment';
 
-export default Component.extend(EventMixin, {
+export default Component.extend({
```

Comment:
just double-checking here -we're _not_ extending `EventsBase` on purpose here, right?

Reply:
That's correct. The method we were using from the mixin isn't here anymore and was replaced with `courseId: reads('event.course'),` so it can go away entirely.

Reply:
:+1: